app/views/container_group/_main.html.haml

```diff
@@ -1,7 +1,10 @@
 = render :partial => "layouts/flash_msg"
 .row
   .col-md-12.col-lg-6
-    = render :partial => "shared/summary/textual", :locals => { :title => _("Properties"), :items => textual_group_properties }
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"),
+    :items => textual_group_properties}
```

Comment:
I don't like these options being on a newline, unindented.  Hard for the reader to see that they belong with the previous line.  Either put everything back onto a single line or come up with a way to indent it to make it visually obvious what is going on.